enterprise/internal/batches/types/changeset_spec.go

```diff
@@ -157,11 +157,17 @@ func (cs *ChangesetSpec) computeDiffStat() error {
 
 // computeForkNamespace calculates the namespace that the changeset spec will be
 // forked into, if any.
-func (cs *ChangesetSpec) computeForkNamespace() {
+func (cs *ChangesetSpec) computeForkNamespace(forkFromSpec *bool) {
 	// Right now, we only look at the global enforceForks setting, but we will
 	// likely base this off the description eventually as well.
```

Comment:
Hey look, we did this. 😄

```suggestion
	// If the fork property is unspecified in the batch spec changesetTemplate,
	// we only look at the global enforceForks setting. But if the property *is*
	// specified, it takes precedence.
```